ci/pipelines/pull-request.yml.erb

```diff
@@ -50,6 +50,11 @@ jobs:
     - put: dockerhub-pr
       params:
         build: github-pull-request
+    - put: github-pull-request
```

Comment:
I think this was a bad rebase. This should be reverted.

Reply:
Not sure what happened there, I reverted that last commit.